src/envo/envo-edit.owl

```diff
@@ -37272,6 +37302,57 @@ AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> <ht
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000133> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000133> "nuclear reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000133> <http://purl.obolibrary.org/obo/ENVO_00000070>)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000133> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/RO_0002215> ObjectIntersectionOf(<http://purl.obolibrary.org/obo/ENVO_2000012> ObjectSomeValuesFrom(<http://purl.obolibrary.org/obo/BFO_0000055> <http://purl.obolibrary.org/obo/ENVO_2000025>))))
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000134> (thermal nuclear reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Thermal-neutron_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000134> "a nuclear reactor which uses slow or thermal neutrons.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000134> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000134> "thermal reactor")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000134> "thermal nuclear reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000134> <http://purl.obolibrary.org/obo/ENVO_02000137>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000135> (fast neutron reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Fast-neutron_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000135> "A nuclear reactor in which the fission chain reaction is sustained by fast neutrons.")
+AnnotationAssertion(<http://purl.org/dc/elements/1.1/creator> <http://purl.obolibrary.org/obo/ENVO_02000135> <https://orcid.org/0000-0002-6601-2165>)
+AnnotationAssertion(<http://www.geneontology.org/formats/oboInOwl#hasExactSynonym> <http://purl.obolibrary.org/obo/ENVO_02000135> "fast reactor")
+AnnotationAssertion(rdfs:label <http://purl.obolibrary.org/obo/ENVO_02000135> "fast neutron reactor"@en)
+SubClassOf(<http://purl.obolibrary.org/obo/ENVO_02000135> <http://purl.obolibrary.org/obo/ENVO_02000137>)
+
+# Class: <http://purl.obolibrary.org/obo/ENVO_02000136> (research nuclear reactor)
+
+AnnotationAssertion(Annotation(<http://www.geneontology.org/formats/oboInOwl#hasDbXref> "https://en.wikipedia.org/wiki/Research_reactor") <http://purl.obolibrary.org/obo/IAO_0000115> <http://purl.obolibrary.org/obo/ENVO_02000136> "A nuclear reactor which serves primarily as a neutron source.")
```

Comment:
The definition doesn't match the label. There are more uses for neutron sources than simply research.

Reply:
revised definition and provided more sources. We could change the primary label to non-power but research appears to be the term used

Reply:
Okay, the exact synonyms help too. I think that works, although I do think more accurate labels are better.